src/cloud-cli/userdb.ts

```diff
@@ -76,6 +91,19 @@ export async function deleteUserDb(host: string, port: string, dbName: string, s
         // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
           status = data.Status
       }
+
+      // Update the clouddb info record
+      logger.info("Saving db state to cloud db");
```

Comment:
We'll have to revisit our user-facing logs :-) This seems useful for debugging but I don't think users will really understand this line nor get value from it.